stack/Chart.yaml

```
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.6.0
+version: 1.6.1
```

```suggestion
version: 1.6.0
```

Since the real fix is already merged, just add a comment here like "#bump" and fix the title of the pr